tap_github/client.py

```diff
@@ -71,13 +72,61 @@ def get_url_params(
                 params["since"] = since
         return params
 
+    def _request_with_backoff(
```

Comment:
We never call the super (from what I see) which I'm guessing means we copy a lot of logic from the parent. Probably better to try change the underlying impl via a fork or message to the maintainer? Or maybe we can call the super and add extra code as needed.